app/controllers/samples_controller.rb

```diff
@@ -157,7 +157,7 @@ def show
     align_summary_file = @pipeline_run ? "#{@pipeline_run.alignment_viz_output_s3_path}.summary" : nil
     @align_viz = true if params[:align_viz] && align_summary_file && get_s3_file(align_summary_file)
 
-    if @pipeline_run && (@pipeline_run.remaining_reads.to_i > 0 || @pipeline_run.finalized?) && !@pipeline_run.failed?
+    if @pipeline_run && (((@pipeline_run.remaining_reads.to_i > 0 || @pipeline_run.finalized?) && !@pipeline_run.failed?) || @pipeline_run.report_ready?)
```

Comment:
This is getting harder to digest :)  Does it mean "At least one stage has completed successfully and stage 2 has not failed"?   We need to normalize our status flags so this is easier to express.